setup.py

```diff
@@ -0,0 +1,25 @@
+import os
+from setuptools import setup, find_packages
+
+here = os.path.abspath(os.path.dirname(__file__))
+
+setup(
+  name="schemas.zcrypto",
```

Comment:
It's probably good not to have a period in a package name given that's how you go from a package to a module.

Reply:
gotcha. I had it set up to match the package name, but that doesn't seem to be necessary. I'll change this to `zcypto-schema` (and `zgrab2-schema`).